### PR TITLE
Add more service names in donet, python, js and java csv files

### PR DESCRIPTION
**Pull request by v-xuto**
1. Update names and add more service names in donet, python, js and java csv files.
2. Remove seven Android packages in java-packages.csv 

@jongio for notification

### CODE REVIEW

**Comment by jongio** [comment]
@weshaggard  - Can you please review?

File: _data/releases/latest/dotnet-packages.csv [hunk 1]
```diff
@@ -10,7 +10,7 @@
 "Azure.Communication.Common","","1.0.0-beta.3","Communication Common","Communication","communication","","","client","true","",""
 "Azure.Communication.Sms","","1.0.0-beta.3","Communication SMS","Communication","communication","","","client","true","",""
 "Azure.Core","1.6.0","","Core","Core","core","","","client","true","",""
-"Azure.Core.Amqp","1.0.0","","Core - AMQP","Core","core","NA","","client","true","",""
+"Azure.Core.Amqp","1.0.0","","Core AMQP","Core","core","NA","","client","true","",""
```
Comment (weshaggard) [inline _data/releases/latest/dotnet-packages.csv:13]: Other languages like Java call this "Core - AMQP" why the change here?

File: _data/releases/latest/dotnet-packages.csv [hunk 2]
```diff
@@ -440,43 +440,43 @@
 "Microsoft.Azure.Jobs.Core","","0.3.2-beta","Jobs - Core","WebJobs","https://github.com/Azure/azure-webjobs-sdk","NA","NA","client","false","",""
 "Microsoft.Azure.Jobs.ServiceBus","","0.3.2-beta","Jobs - Service Bus","WebJobs","https://github.com/Azure/azure-webjobs-sdk","NA","NA","client","false","",""
 "Microsoft.Azure.KeyVault","3.0.5","","Key Vault","Key Vault","keyvault","NA","","client","false","","Replaced By: Azure.Security.KeyVault.Keys<br>Azure.Security.KeyVault.Secrets<br>Azure.Security.KeyVault.Certificates"
-"Microsoft.Azure.KeyVault.Core","3.0.5","","Key Vault - Core","Storage","keyvault","NA","","client","false","","Replaced By: Azure.Security.KeyVault.Keys<br>Azure.Security.KeyVault.Secrets<br>Azure.Security.KeyVault.Certificates"
```
Comment (weshaggard) [inline _data/releases/latest/dotnet-packages.csv:443]: We intentionally put this under the Storage service name. See https://github.com/Azure/azure-sdk/pull/2019

**Comment by weshaggard** [comment]
@v-xuto What process are you using to ensure the names are consistent across the different languages? It might be worth adding some level of validation to ensure the names match across languages.

File: _data/releases/latest/python-packages.csv [hunk 3]
```diff
@@ -140,82 +140,137 @@
...
+"azure-mgmt-synapse","0.4.0","","Resource Management - Synapse","Azure Synapse Analytics","synapse","","","mgmt","false","",""
 "azure-mgmt-timeseriesinsights","0.2.0","","Resource Management - Time Series Insights","Time Series Insights","timeseriesinsights","NA","","mgmt","false","",""
 "azure-mgmt-trafficmanager","0.51.0","","Resource Management - Traffic Manager","Traffic Manager","https://github.com/Azure/azure-sdk-for-python/tree/azure-mgmt-trafficmanager_0.51.0/azure-mgmt-trafficmanager","https://docs.microsoft.com/python/api/overview/azure/traffic-manager","","mgmt","false","",""
-"azure-mgmt-vmwarecloudsimple","0.2.0","","Resource Management - VM Ware Cloud Simple","Compute","compute","NA","","mgmt","false","",""
-"azure-servicemanagement-legacy","0.20.7","","Service Management Legacy","Core","core","NA","NA","mgmt","false","",""
+"azure-mgmt-vmwarecloudsimple","0.2.0","","Resource Management - VM Ware Cloud Simple","VMWare by CloudSimple","compute","NA","","mgmt","false","",""
+"azure-servicemanagement-legacy","0.20.7","","Service Management - Legacy","Core","core","NA","NA","mgmt","false","",""
+"","","","","Advanced Threat Protection","","","","","","true",""
```
Comment (weshaggard) [inline _data/releases/latest/python-packages.csv:222]: What are these new entries?

File: _data/releases/latest/dotnet-packages.csv [hunk 4]
```diff
@@ -725,60 +725,99 @@
...
 "Microsoft.WindowsAzure.Management.Store","","0.15.1-preview","WindowsAzure Management - Store","Store","NA","NA","NA","mgmt","false","",""
 "Microsoft.WindowsAzure.Management.StorSimple","","1.0.5-preview","WindowsAzure Management - StorSimple","StorSimple","NA","NA","NA","mgmt","false","",""
 "Microsoft.WindowsAzure.Management.TrafficManager","","0.16.1-preview","WindowsAzure Management - Traffic Manager","Traffic Manager","NA","NA","NA","mgmt","false","",""
-"Microsoft.WindowsAzure.Management.VirtualNetworks","","0.9.9-preview","WindowsAzure Management - Virtual Networks","Virtual Networks","NA","NA","NA","mgmt","false","",""
-"Microsoft.WindowsAzure.Management.WebSites","","5.0.0-prerelease","WindowsAzure Management - WebSites","WebSites","NA","NA","NA","mgmt","false","",""
+"Microsoft.WindowsAzure.Management.VirtualNetworks","","0.9.9-preview","WindowsAzure Management - Virtual Networks","Virtual Network","NA","NA","NA","mgmt","false","",""
+"Microsoft.WindowsAzure.Management.WebSites","","5.0.0-prerelease","WindowsAzure Management - WebSites","App Service","NA","NA","NA","mgmt","false","",""
+"","","","","Advanced Threat Protection","","","","","","true",""
```
Comment (weshaggard) [inline _data/releases/latest/dotnet-packages.csv:785]: What are these extra entries?

File: _data/releases/latest/java-packages.csv [hunk 5]
```diff
@@ -730,30 +723,90 @@
...
+"azure-svc-mgmt-media","com.microsoft.azure","0.9.8","","SVC Management - Media","Media Services","NA","NA","NA","mgmt","false","TRUE",""
+"azure-svc-mgmt-network","com.microsoft.azure","0.9.8","","SVC Management - Network","Virtual Network","NA","NA","NA","mgmt","false","TRUE",""
 "azure-svc-mgmt-scheduler","com.microsoft.azure","0.9.8","","SVC Management - Scheduler","Scheduler","NA","NA","NA","mgmt","false","TRUE",""
-"azure-svc-mgmt-sql","com.microsoft.azure","0.9.8","","SVC Management - SQL","SQL","NA","NA","NA","mgmt","false","TRUE",""
+"azure-svc-mgmt-sql","com.microsoft.azure","0.9.8","","SVC Management - SQL","Azure SQL","NA","NA","NA","mgmt","false","TRUE",""
 "azure-svc-mgmt-storage","com.microsoft.azure","0.9.8","","SVC Management - Storage","Storage","NA","NA","NA","mgmt","false","TRUE",""
-"azure-svc-mgmt-websites","com.microsoft.azure","0.9.8","","SVC Management - Websites","Websites","NA","NA","NA","mgmt","false","TRUE",""
+"azure-svc-mgmt-websites","com.microsoft.azure","0.9.8","","SVC Management - Websites","App Service","NA","NA","NA","mgmt","false","TRUE",""
+"","","","","Advanced Threat Protection","","","","","","","true",""
```
Comment (weshaggard) [inline _data/releases/latest/java-packages.csv:753]: What are these extra entries?

File: _data/releases/latest/js-packages.csv [hunk 6]
```diff
@@ -406,53 +406,53 @@
...
-"azure-streamanalytics-cicd","2.0.0","","Stream Analytics CI/CD","Stream Analytics","NA","NA","NA","client","false","",""
-"@azure/swagger-validation-common","0.1.2","","Swagger Validation Common","Swagger Validation ","NA","NA","NA","client","false","",""
-"@azure/synapse","","1.0.0-preview.1","Synapse","Synapse Analytics","https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/synapse/synapse","NA","NA","client","false","",""
-"@azure/template","","0.2.0-preview.4","Template","Template","","NA","NA","client","false","true",""
+"azure-streamanalytics-cicd","2.0.0","","Stream Analytics CI/CD","Azure Stream Analytics","NA","NA","NA","client","false","",""
+"@azure/swagger-validation-common","0.1.2","","Swagger Validation Common","OpenAPI Specification","NA","NA","NA","client","false","",""
+"@azure/synapse","","1.0.0-preview.1","Synapse","Azure Synapse Analytics","https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/synapse/synapse","NA","NA","client","false","",""
+"@azure/template","","0.2.0-preview.4","Template","Templates","","NA","NA","client","false","true",""
```
Comment (weshaggard) [inline _data/releases/latest/js-packages.csv:438]: I would keep this as Template

File: _data/releases/latest/dotnet-packages.csv [hunk 7]
```diff
@@ -487,24 +487,24 @@
...
-"Microsoft.Azure.Synapse","","0.1.0-preview","Synapse","Synapse Analytics","synapse","NA","","client","false","",""
+"Microsoft.Azure.Synapse","","0.1.0-preview","Synapse","Azure Synapse Analytics","synapse","NA","","client","false","",""
 "Microsoft.Azure.Cosmos.Table","","2.0.0-preview","Tables","Tables","NA","NA","NA","client","false","",""
 "Microsoft.Azure.CosmosDB.Table","2.1.2","","Tables","Tables","NA","NA","NA","client","false","","Will be replaced by: Azure.Data.Tables"
-"Azure.Template","","1.0.3-beta.20201117","Template","Template","NA","","NA","client","false","true",""
-"Microsoft.Azure.Template","","1.0.2-preview1","Template","Template","NA","NA","NA","client","false","",""
-"Microsoft.Azure.Test.HttpRecorder","1.13.3","","Test HttpRecorder","Test HttpRecorder","https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/mgmtcommon/TestFramework/Microsoft.Azure.Test.HttpRecorder","NA","NA","client","false","",""
+"Azure.Template","","1.0.3-beta.20201117","Templates","Templates","NA","","NA","client","false","true",""
```
Comment (weshaggard) [inline _data/releases/latest/dotnet-packages.csv:493]: I would keep this as Template

File: _data/releases/latest/java-packages.csv [hunk 8]
```diff
@@ -280,153 +273,153 @@
...
-"azure-schemaregistry-kafka-avro","com.microsoft.azure","","1.0.0-beta.4","Schema Registry Kafka Avro","Schema","NA","NA","NA","client","false","",""
-"azure-sdk-template","com.azure","1.1.0","1.2.2-beta.1","SDK - Template","Template","template","","","client","false","true",""
-"azure-sdk-template","com.microsoft.azure","1.0.1","","SDK - Template","Template","template","","NA","client","false","true",""
-"azure-sdk-template-bom","com.azure","","1.0.0-beta.5","SDK - Template BOM","Template","","NA","NA","client","false","true",""
+"qpid-proton-j-extensions","com.microsoft.azure","1.2.3","","Qpid Proton J Extensions","Extensions on qpid-proton-j library","NA","NA","NA","client","false","",""
+"azure-relay","com.microsoft.azure","0.0.3","","Relay","Azure Relay","NA","NA","NA","client","false","",""
+"azure-schemaregistry-kafka-avro","com.microsoft.azure","","1.0.0-beta.4","Event Hubs - Schema Registry Kafka Avro","Event Hubs","NA","NA","NA","client","false","",""
+"azure-sdk-template","com.azure","1.1.0","1.2.2-beta.1","SDK - Template","Templates","template","","","client","false","true",""
```
Comment (weshaggard) [inline _data/releases/latest/java-packages.csv:310]: I would keep these as Template.

File: _data/releases/latest/python-packages.csv [hunk 9]
```diff
@@ -101,35 +101,35 @@
...
-"azure-synapse-artifacts","0.3.0","","Synapse Artifacts","Synapse Analytics","synapse","","","client","false","",""
-"azure-synapse-nspkg","1.0.0","","Synapse Namespace Package","Synapse Analytics","synapse","NA","NA","client","false","",""
-"azure-synapse-spark","0.3.0","","Synapse Spark","Synapse Analytics","synapse","","","client","false","",""
-"azure-template","0.0.17","0.1.0b626355","Template","Template","template","NA","","client","false","true",""
+"azure-synapse","0.1.0","","Synapse","Azure Synapse Analytics","synapse","NA","","client","false","",""
+"azure-synapse-accesscontrol","0.3.0","","Synapse - Access Control","Azure Synapse Analytics","synapse","","","client","false","",""
+"azure-synapse-artifacts","0.3.0","","Synapse - Artifacts","Azure Synapse Analytics","synapse","","","client","false","",""
+"azure-synapse-nspkg","1.0.0","","Synapse Namespace Package","Azure Synapse Analytics","synapse","NA","NA","client","false","",""
+"azure-synapse-spark","0.3.0","","Synapse - Spark","Azure Synapse Analytics","synapse","","","client","false","",""
+"azure-template","0.0.17","0.1.0b626355","Template","Templates","template","NA","","client","false","true",""
```
Comment (weshaggard) [inline _data/releases/latest/python-packages.csv:109]: I would keep this as Template

File: _data/releases/latest/js-packages.csv [hunk 10]
```diff
@@ -462,140 +462,194 @@
...
 "@azure/arm-visualstudio","2.2.0","","Resource Management - Visual Studio","Visual Studio","https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/visualstudio/arm-visualstudio","NA","","mgmt","false","",""
-"@azure/arm-vmwarecloudsimple","1.0.0","","Resource Management - VMware CloudSimple","Vmware CloudSimple","https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/vmwarecloudsimple/arm-vmwarecloudsimple","NA","","mgmt","false","",""
+"@azure/arm-vmwarecloudsimple","1.0.0","","Resource Management - VMware CloudSimple","VMWare by CloudSimple","https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/vmwarecloudsimple/arm-vmwarecloudsimple","NA","","mgmt","false","",""
 "@azure/arm-webservices","0.1.1","","Resource Management - Web Services","Machine Learning","https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/machinelearning/arm-webservices","NA","","mgmt","false","",""
 "@azure/arm-workspaces","0.1.1","","Resource Management - Work Spaces","Machine Learning","https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/machinelearning/arm-workspaces","NA","","mgmt","false","",""
+"","","","","Advanced Threat Protection","","","","","","true",""
```
Comment (weshaggard) [inline _data/releases/latest/js-packages.csv:602]: Extra empty entries.

File: _data/releases/latest/java-packages.csv [hunk 11]
```diff
@@ -207,11 +200,11 @@
 "azure-cosmosdb-spring-boot-starter","com.microsoft.azure","2.3.5","","Cosmos DB - Spring Boot Starter",Spring Boot,"NA","","NA","client","false","",""
```
Comment (yiliuTo) [inline _data/releases/latest/java-packages.csv:200]: Hi @v-xuto , thanks for updating the Spring packages. But it seems not all of the packages with ServiceName of "Spring Boot" or "Spring Cloud" have been added the prefix of "Azure" on their ServiceNames, so could you like to have a check and update them?

Comment (v-xuto) [reply _data/releases/latest/java-packages.csv:200]: @yiliuTo - Thank you for your reminder, I will double check and update.

Comment (weshaggard) [reply _data/releases/latest/java-packages.csv:200]: We should avoid adding "Azure" as a prefix in the names as that is assumed in all the context we use these names.

Comment (v-xuto) [reply _data/releases/latest/java-packages.csv:200]: @weshaggard - Thanks for your advice. I will remove the "Azure" prefix from these names.

File: _data/releases/latest/java-packages.csv [hunk 12]
```diff
@@ -151,30 +144,30 @@
...
 "autorest-clientruntime-for-java","com.microsoft.azure.v2","2.1.1","","AutoRest Client Runtime for Java","AutoRest","NA","NA","NA","client","false","",""
 "azure-cognitiveservices-autosuggest","com.microsoft.azure.cognitiveservices","","1.0.2-beta","Autosuggest","Autosuggest","https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/cognitiveservices/ms-azure-cs-autosuggest","NA","NA","client","false","",""
 "azure","com.microsoft.azure","1.37.1","","Azure","Azure","NA","NA","NA","client","false","",""
-"azure-active-directory-b2c-spring-boot-starter","com.microsoft.azure","2.3.5","","Azure Active Directory - B2C Spring Boot Starter",Spring Boot,"https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/spring/azure-spring-boot-starter-active-directory-b2c","","NA","client","false","",""
+"azure-active-directory-b2c-spring-boot-starter","com.microsoft.azure","2.3.5","","Azure Active Directory B2C Spring Boot Starter","Azure Active Directory B2C","https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/spring/azure-spring-boot-starter-active-directory-b2c","","NA","client","false","",""
```
Comment (yiliuTo) [inline _data/releases/latest/java-packages.csv:155]: @v-xuto , could you help to restore ServiceName of `azure-active-directory-b2c-spring-boot-starter` as `Spring Boot`?

File: _data/releases/latest/java-packages.csv [hunk 13]
```diff
@@ -151,30 +144,30 @@
...
 "azure-client-authentication","com.microsoft.azure","1.7.10","","Client Authentication","Authentication","NA","NA","NA","client","false","",""
 "azure-client-authentication","com.microsoft.azure.v2","","2.0.0-java7-beta1","Client Authentication","Authentication","NA","NA","NA","client","false","true",""
 "azure-client-runtime","com.microsoft.azure","1.7.10","","Client Runtime","AutoRest","NA","NA","NA","client","false","",""
 "azure-client-runtime","com.microsoft.azure.v2","","2.0.0-java7-beta1","Client Runtime","AutoRest","NA","NA","NA","client","false","true",""
 "client-runtime","com.microsoft.azure","1.0.1","","Client Runtime","AutoRest","NA","NA","NA","client","false","true",""
-"azure-cloud-foundry-service","com.microsoft.azure","0.1.7","","Cloud Foundry Service","Cloud Foundry","NA","NA","NA","client","false","",""
-"azure-cognitiveservices-parent","com.microsoft.azure.cognitiveservices","1.0.2","","Cognitive Services - Parent","Parent","NA","NA","NA","client","false","",""
+"azure-cloud-foundry-service","com.microsoft.azure","0.1.7","","Cloud Foundry Service","Azure Spring Boot","NA","NA","NA","client","false","",""
```
Comment (yiliuTo) [inline _data/releases/latest/java-packages.csv:169]: And the ServiceName here is Spring Cloud.

File: _data/releases/latest/java-packages.csv (hunk 12 above)
Comment (v-xuto) [reply _data/releases/latest/java-packages.csv:155]: @yiliuTo - Sure, I will update it.

File: _data/releases/latest/dotnet-packages.csv (hunk 1 above)
Comment (v-xuto) [reply _data/releases/latest/dotnet-packages.csv]: Updated.

File: _data/releases/latest/dotnet-packages.csv (hunk 2 above)
Comment (v-xuto) [reply _data/releases/latest/dotnet-packages.csv]: Updated.

File: _data/releases/latest/js-packages.csv (hunk 6 above)
Comment (v-xuto) [reply _data/releases/latest/js-packages.csv]: Updated.

File: _data/releases/latest/java-packages.csv (hunk 11 above)
Comment (v-xuto) [reply _data/releases/latest/java-packages.csv]: I have removed the "Azure" prefix from these names.

File: _data/releases/latest/java-packages.csv (hunk 12 above)
Comment (v-xuto) [reply _data/releases/latest/java-packages.csv]: Updated.

Comment (yiliuTo) [reply _data/releases/latest/java-packages.csv]: Thanks~

File: _data/releases/latest/dotnet-packages.csv (hunk 1 above)
Comment (v-xuto) [reply _data/releases/latest/dotnet-packages.csv]: @weshaggard - We plan to add missing packages. At present, add their corresponding service names first, and then further search for and add packages corresponding to these service names.

File: _data/releases/latest/dotnet-packages.csv (hunk 2 above)
Comment (v-xuto) [reply _data/releases/latest/dotnet-packages.csv]: Same as above.

File: _data/releases/latest/python-packages.csv (hunk 3 above)
Comment (v-xuto) [reply _data/releases/latest/python-packages.csv:225]: Same as above

File: _data/releases/latest/dotnet-packages.csv (hunk 4 above)
Comment (v-xuto) [reply _data/releases/latest/dotnet-packages.csv:787]: Same as above.

File: _data/releases/latest/java-packages.csv (hunk 5 above)
Comment (v-xuto) [reply _data/releases/latest/java-packages.csv]: Same as above.

File: _data/releases/latest/js-packages.csv (hunk 10 above)
Comment (v-xuto) [reply _data/releases/latest/js-packages.csv:604]: Same as above.

File: _data/releases/latest/dotnet-packages.csv (hunk 7 above)
Comment (v-xuto) [reply _data/releases/latest/dotnet-packages.csv]: Updated.

File: _data/releases/latest/java-packages.csv (hunk 8 above)
Comment (v-xuto) [reply _data/releases/latest/java-packages.csv]: Updated.

File: _data/releases/latest/python-packages.csv (hunk 9 above)
Comment (v-xuto) [reply _data/releases/latest/python-packages.csv]: Updated.

File: _data/releases/latest/java-packages.csv [hunk 14]
```diff
@@ -63,16 +63,16 @@
...
-"azure-core-tracing-opentelemetry","com.azure","","1.0.0-beta.6","Tracing","Core","core","","","client","true","",""
-"tracing-opentelemetry","com.azure","","1.0.0-preview.2","Tracing - OpenTelemetry","Tracing","tracing","NA","NA","client","true","",""
-"troposphere-maven-plugin","com.azurenight.maven","2.6.0.1","","Troposphere Maven Plugin","Troposphere","NA","NA","NA","client","true","",""
+"azure-core-tracing-opentelemetry","com.azure","","1.0.0-beta.6","Tracing OpenTelemetry Plugin","Plugin","core","","","client","true","",""
+"tracing-opentelemetry","com.azure","","1.0.0-preview.2","Tracing OpenTelemetry Plugin","Plugin","tracing","NA","NA","client","true","true",""
+"troposphere-maven-plugin","com.azurenight.maven","2.6.0.1","","Troposphere Maven Plugin","Plugin","NA","NA","NA","client","true","",""
 "azure-core-management","com.azure","1.0.0","","Management - Core","Core","core","","","mgmt","true","",""
-"azure-resourcemanager","com.azure.resourcemanager","2.0.0","","Resource Management","Mangement","NA","","","mgmt","true","",""
+"azure-resourcemanager","com.azure.resourcemanager","2.0.0","","Resource Management","Resources","NA","","","mgmt","true","",""
```
Comment (weshaggard) [inline _data/releases/latest/java-packages.csv:70]: @nickzhums what service name did you want this track 2 mgmt library under?

**Review by weshaggard — APPROVED** [review]
